packages/replay-next/components/sources/PointPanel.module.css

```
@@ -69,6 +69,10 @@
   gap: 1ch;
 }
 
+.ContentWrapper:hover {
+  border: 2px solid var(--point-panel-input-border-color-hover);
```

I like following this approach for styles like this (so we don't see any unexpected layout shift or clipping as a result of the hover):
```css
.ClassName {
  border: 2px solid transparent;
}
.ClassName:hover {
  border-color: var(--whatever);
}
```

It looks like this component _does_ follow that pattern– but redeclaring the whole `border` shorthand still _could_ make it easier for the two to fall out of sync, so small suggested change:
```suggestion
  border-color: var(--point-panel-input-border-color-hover);
```

---

Good call, thanks!